src/modules/Auth/components/Register/Register.tsx

```diff
@@ -384,52 +367,72 @@ export default function Register() {
                 >
                   Profile Image
                 </label>
-                <input
-                  type="file"
-                  id="profileImage"
-                  {...register("profileImage")}
-                /> */}
-
+                <Button
+                  component="label"
+                  role={undefined}
+                  tabIndex={-1}
+                  variant="outlined"
+                  sx={{
+                    textTransform: "none",
+                    marginBottom: 1,
+                    bgcolor: "#f5f6f8",
+                    border: "none",
+                    padding: 1.5,
+                  }}
+                >
```

Comment:
It is a good practice to show the user a preview for the image before submitting it. We can do this using createObjectURL method. You can find more about it here:
https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL_static